tests/unit/amo/components/TestInstallButtonWrapper.js

```diff
@@ -504,19 +504,4 @@ describe(__filename, () => {
 
     expect(root.find('.InstallButtonWrapper-download')).toHaveLength(0);
   });
-
-  it('does not display a download link when currentVersion has no file', () => {
-    const _getClientCompatibility = sinon.stub().returns({
-      compatible: false,
-    });
-    const root = render({
-      _getClientCompatibility,
-      version: createInternalVersionWithLang({
-        ...fakeAddon.current_version,
-        files: [],
-      }),
-    });
-
-    expect(root.find('.InstallButtonWrapper-download')).toHaveLength(0);
-  });
```

Comment:
Why is this test case removed?

Reply:
because versions can't be a state where no file exists